schema/switch.yml

```diff
@@ -242,3 +242,294 @@ properties:
+          type: boolean
+          default: false
+          enum: 
+            - true
+            - false
+        dhcp-snoop-inf-opt-encode-subopt:
+          description: "This parameter allows for the encoding of sub-options within option 82 to further specify client information"
+          type: boolean
+          default: false
+          enum: 
+            - true
+            - false
+        dhcp-snoop-inf-opt-remoteid:
+          description: "It specifies the remote ID sub-option in option 82, which typically includes information like the circuit ID or remote host identifier"
+          type: string
+          maxLength: 32
+          minLength: 1
+        dhcp-snoop-inf-opt-policy:
+          description: "This defines the policy for handling packets with option 82, determining whether they should be forwarded or dropped based on the configuration"
+          type: string
+          enum: 
+            - drop
+            - keep
+            - replace
+  acl:
+    description: Contains all the access control rule definitions
+    type: array
+    items:
+      type: object
+      properties:
+        acl-type:
+          description: Type of the access control list
+          type: string
+          enum:
+            - ipv4
+            - ipv6
+            - ipv4Ext
+            - ipv6Ext
+            - mac
+            - arp
+        acl-name:
+          description: The identifier or name for the Access Control List
+          type: string
+          maxLength: 32
+          minLength: 1
+        acl-rule-action:
+          description: Defines whether to permit or deny traffic matching the rule
+          type: string
+          enum:
+            - permit
+            - deny
+        acl-source-macaddress:
+          description: Specifies the source MAC address to filter on
+          type: string
+          format: uc-mac
+        acl-source-macbitmask:
+          description: The mask applied to the source MAC address
+          type: string
+          format: uc-mac
+        acl-dest-macaddress:
+          description: Specifies the destination MAC address for the filter
+          type: string
+          format: uc-mac
+        acl-dest-macbitmask:
+          description: The mask applied to the destination MAC address
+          type: string
+          format: uc-mac
+        acl-packet-format:
+          description: Identifies the protocol encapsulated in the Ethernet frame by its EtherType
+          type: string
+          enum:
+            - any
+            - untagged-eth2
+            - untagged-802.3
+            - tagged-eth2
+            - tagged-802.3
+        acl-vlanid:
+          description: Specifies a VLAN ID to filter traffic from a specific VLAN
+          type: integer
+          minimum: 1
+          maximum: 4094
```

Comment:
I do understand that 4095 is reserved, but i don't think we excluded 4095 in the schema anywhere else;
Should we maybe change any other occurances of VID to exclude 4095, or just let 4095 be allowed here as well?

Reply:
It was specified in the guest-vlan  & unauthenticated-vlan existing definition from ethernet.yml. I guess in other places that i could see there is no min/max specified. The way i interpret these min and max are for validation rules so as to not let any vlan out of this range allowed in.